root/dataframe/CMakeLists.txt

```diff
@@ -235,10 +235,18 @@ ROOTTEST_ADD_TEST(read_leaves_nodict
                   EXEC ./read_leaves_nodict
                   DEPENDS ${GENERATE_EXECUTABLE_TEST} read_leaves)
 
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+# In debug mode RDataFrame warns when it needs to copy collection elements for reading
+ROOTTEST_ADD_TEST(test_readTotemNtuple
```

Comment:
To focus on the difference, I would make a variable to hold the reference filename rather than duplicating the full `ROOTTEST_ADD_TEST`.

Reply:
Another option (if the message issue during debug is not important) is to use an `OUTCNVCMD` to filter out the unwanted line.